packages/headless/src/app.module.ts

```diff
@@ -1,19 +1,14 @@
-import { Module, MiddlewareConsumer } from "@nestjs/common";
-import { HelperModule } from "./helper/helper.module";
-import { LoggerMiddleware } from "./middleware/logger.middleware";
-import { ProductModule } from "./modules/product/product.module";
+import { Module, MiddlewareConsumer } from '@nestjs/common';
+import { HelperModule } from './helper/helper.module';
+import { LoggerMiddleware } from './middleware/logger.middleware';
+import { CartModule } from './modules/cart/cart.module';
+import { ProductModule } from './modules/product/product.module';
 
 @Module({
-  imports: [
-    ProductModule,
-    HelperModule
-  ]
+  imports: [ProductModule, CartModule, HelperModule],
```

Comment:
multi-line is better here. because for lots of module single line would not be affordable

Reply:
done